### DISCUSSION TITLE
Dollar sign backtick

### DISCUSSION THREAD

**Discussion by dwhall** [PDConSec/vsc-print#347] [General]
Thanks for the print plugin.  It works well for me most of the time.  I think I've found an issue, though.  Anytime the two characters (dollar sign)(backtick) or "$`" appear in the code, the output is wonky.  Those two characters rarely appear naturally in languages like C and Python.  But if you take any file type and add those two characters, you can recreate the issue.
[timmy.py.pdf](https://github.com/user-attachments/files/16950060/timmy.py.pdf)

**Comment by PeterWone** [comment]
This relates to a problem I was already investigating, and I suspect you just make it a lot easier to fathom. I think I know what's going on now. Thanks

**Comment by PeterWone** [comment]
I brought forward some maintenance work and instead of ad hoc token replacement I'm using handlebars. It seems to have resolved the issue (not released yet). See #343

**Comment by PeterWone** [comment]
@dwhall The current release should fix this. Can you confirm that this is now working for you?

**Reply by dwhall** [reply]
Yes, it works.  Thanks a bunch!  Nice turnaround.
